src/serac/infrastructure/initialize.cpp

```diff
@@ -95,7 +95,8 @@ std::pair<int, int> initialize(int argc, char* argv[], MPI_Comm comm)
   profiling::initializeCaliper();
 
   // Initialize GPU (no-op if not enabled/available)
-  accelerator::initializeDevice();
+  // TODO for some reason this causes errors on Lassen. We need to look into this ASAP.
+  // accelerator::initializeDevice();
```

Comment:
The lassen builds are currently broken due to a device initialization call. We need to figure out the root cause and re-enable this.